src/main/java/com/kodilla/ecommercee/ProductController.java

```diff
@@ -1,4 +1,36 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.web.bind.annotation.*;
+
+import javax.transaction.Transactional;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@RestController
+@RequestMapping("v1/ecommercee")
+@Transactional
 public class ProductController {
+
+    @RequestMapping(method = RequestMethod.GET, value = "getAllProducts", consumes = "application/json")
```

Comment:
Make it more readeable by using @GetMapping instead
```suggestion
    @GetMapping(value = "getAllProducts", consumes = "application/json")
```